database/layer/admin-client.js

```diff
@@ -69,21 +64,35 @@ async function insertClient (tenantName, clientId, name, description, secret) {
   }
 }
 
-async function updateClientByClientId (clientId, name, description, secret) {
-  const sql = 'UPDATE usher.clients SET name = $1, description = $2, secret = $3 WHERE client_id = $4'
+/**
+ * Updates a client by client ID with the provided information.
+ *
+ * @param {string} clientId - The ID of the client to update.
+ * @param {Object} clientInfo - Object containing the updated client information.
+ * @param {string} clientInfo.client_id - The new client ID of the client.
+ * @param {string} clientInfo.name - The new name of the client.
+ * @param {string} clientInfo.description - The new description of the client.
+ * @param {string} clientInfo.secret - The new secret key of the client.
+ * @returns {Promise<Object>} The updated client object.
+ * @throws {Error} If an error occurs while updating the client.
+ */
+const updateClientByClientId = async (clientId, { client_id, name, description, secret }) => {
   try {
-    const results = await pool.query(sql, [name, description, secret, clientId])
-    if (results.rowCount === 1) {
-      return 'Update successful'
-    } else {
-      return `Update failed: Client does not exist matching client_id ${clientId}`
-    }
-  } catch (error) {
-    return `Update failed: ${error.message}`
+    const [updatedClient] = await usherDb('clients')
+      .where({ client_id: clientId })
+      .update({
```

Comment:
@k1ch super trivial, but assuming the `clients` table has an `updated_at` column on it, should we also update that field to `now()` ?

Reply:
Good point, yes I will update that.